python/ray/experimental/data/read_api.py

```diff
@@ -17,7 +17,7 @@
 def autoinit_ray(f):
     def wrapped(*a, **kw):
         if not ray.is_initialized():
-            ray.client.connect()
+            ray.client().connect()
```

Comment:
Hmm interesting. We also should have a test for this. Do you know how the current client things are tested?